kratos/python/add_properties_to_python.cpp

```diff
@@ -147,10 +149,39 @@ bool HasTableHelperFunction1( TContainerType& rContainer,
     return rContainer.HasTable(XVar, YVar);
 }
 
+template<typename TVariableType>
+void AddInterfaceToAccessorFold(pybind11::class_<Properties, Properties::Pointer, Properties::BaseType>& pyProperties) {
+
+    pyProperties
+    .def("GetAccessor", [](Properties &rProperties, Variable<TVariableType> &rVariable) { 
+            auto accessor = &rProperties.pGetAccessor(rVariable);
+            
+            if (*accessor == nullptr)
+                KRATOS_ERROR << "Trying to get a consumed or invalid Accessor." << std::endl;
+                
+            return accessor; 
+        }, py::return_value_policy::reference_internal)
+    .def("SetAccessor", [](Properties &rProperties, Variable<TVariableType> &rVariable, std::unique_ptr<Accessor>& rAccessor) {
+            if (rAccessor == nullptr)
+                KRATOS_ERROR << "Trying to set a consumed or invalid Accessor. Accessors are unique. Please create a different one." << std::endl;
+
+            rProperties.SetAccessor(rVariable, std::move(rAccessor));
```

Comment:
As we're moving in here, wouldn't this end with invalid variables in the Python side?

Reply:
Yep. This is why I put all the new checks for nullptr.

Ideally I would create an interface in python that makes the movement transparent by replacing the binding with the one that SetAccessor returned, but its not possible because the binding is immutable. 

I can't manipulate the reference argument in C++ either.

If someone knows how to do it I will happily code it.